docs/modules/ROOT/pages/servlet/authorization/architecture.adoc

```diff
@@ -257,6 +257,7 @@ static RoleHierarchy roleHierarchy() {
         .role("ADMIN").implies("STAFF")
         .role("STAFF").implies("USER")
         .role("USER").implies("GUEST");
```

Comment:
Will you please also fix the stray semi-colon on this line?

Reply:
Totally may bad, thank you!